Gemfile

```diff
@@ -31,40 +33,41 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
-# readline
-gem 'rb-readline'
-
 # Use mysql
 gem 'mysql2'
 
 # Apply bootstrap
 gem 'less-rails'
 
-gem 'twitter-bootstrap-rails', '~> 3.2.2'
+# WORKAROUND: https://github.com/metaskills/less-rails/issues/122
+gem "sprockets", '3.6.3'
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->